hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -147,10 +151,31 @@ public void updateModifcationTime() {
    */
   public void updateLocationInfoList(List<OmKeyLocationInfo> locationInfoList,
       boolean isMpu) {
+    updateLocationInfoList(locationInfoList, isMpu, false);
+  }
+
+  /**
+   * updates the length of the each block in the list given.
+   * This will be called when the key is being committed to OzoneManager.
+   *
+   * @param locationInfoList list of locationInfo
```

Comment:
NIT: can we add all the javadoc for the added params?

Reply:
Is the third parameter added to allow unit test commit arbitrary block id?

Reply:
Sure I will update javadoc. The purpose of `skipBlockIDCheck` third param is to skip the blkLocation check in case of MPU multipart upload overriding case, I have described it with an existing UT [in my below comment](https://github.com/apache/ozone/pull/2108#discussion_r615686509).